jscomp/runtime/js_string.ml

```diff
@@ -152,8 +152,8 @@ external concat : other:t -> t = "concat"
 ]}
 *)
 
-external concatMany : strings:t array -> t = "concat"
-[@@mel.send.pipe: t] [@@mel.variadic]
+external concatMany : sep:t -> t array -> t = "concat"
```

Comment:
I don't think this makes things clearer. Based on the new name, I'd expect `concatMany ~sep:"," [|"bar"; "baz"|] "foo"` to produce `"foo,bar,baz"`, but I would get `"foo,barbaz"` instead.